standard/src/kg/apc/jmeter/reporters/FlexibleFileWriter.java

```diff
@@ -44,7 +44,9 @@
             + "startTimeMillis endTimeMillis "
             + "responseTimeMicros latencyMicros "
             + "requestData responseData responseHeaders "
-            + "threadsCount requestHeaders connectTime";
+            + "threadsCount requestHeaders connectTime "
```

Comment:
The threadsCount field existed but there was no button for it in the GUI.  When adding the button label I used "allThreads" to be consistent with other listeners/plugins but I'll change it to match the internal field name of "threadsCount".